docs/src/cli/configuration.md

```diff
@@ -0,0 +1,71 @@
+# Configuration
+
+The CLI can be configured on a per project basis in the `Cargo.toml` under the `metadata` section.
+
+```toml
+[package.metadata.bevy_cli]
+```
+
+The CLI supports two targets:
+
+- native: `[package.metadata.bevy_cli.native]`
+- web: `[package.metadata.bevy_cli.web]`
+
+For both these targets a release and dev [`profile`] exists that will automatically be chosen by the CLI:
+
+| **Profile Name**         | **Configuration Section**                          |
+|--------------------------|----------------------------------------------------|
+| `release`                | `[package.metadata.bevy_cli.native.release]`       |
+| `dev`                    | `[package.metadata.bevy_cli.native.dev]`           |
+| `web-release`            | `[package.metadata.bevy_cli.web.release]`          |
+| `web`                    | `[package.metadata.bevy_cli.web.dev]`              |
+
+> **Note**
+>
+> The Web profiles inherits from their native counterpart
+
+## Configuration Merging
+
+Configuration is merged in the following order (from general to specific):
+
+1. Base config: `[package.metadata.bevy_cli]`
+2. Target config: `[package.metadata.bevy_cli.{native|web}]`
```

Comment:
I think we also have `package.metadata.bevy_cli.{dev|release}`, we should include them here